internal/utils/kruize/kruize_api.go

```diff
@@ -196,3 +196,32 @@ func Is_valid_recommendation(recommendation kruizePayload.Recommendation, experi
 		return false
 	}
 }
+
+func Delete_experiment_from_kruize(experiment_name string) {
+	url := cfg.KruizeUrl + "/createExperiment"
+	data := []map[string]string{
+		{"experiment_name": experiment_name},
+	}
+	payload, _ := json.Marshal(data)
+
+	client := &http.Client{}
+	req, err := http.NewRequest("DELETE", url, bytes.NewBuffer(payload))
+	if err != nil {
+		kruizeAPIException.WithLabelValues("/deleteExperiment").Inc()
+		log.Error("error occured while deleting experiment: ", err)
+		return
+	}
```

Comment:
Can this be de-duplicated? maybe a separate small func can do the job? or if you wish to keep that inside `Delete_experiment_from_kruize` then var=unamed func and reuse it thrice?

Reply:
Done